### ISSUE TITLE
Re-enable ability to run distinct queries for Taxon

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify7#3159]
See https://github.com/specify/specify7/pull/1019#issuecomment-973525594

Need to fix the back-end bug and then re-enable it on the front-end

**Comment by grantfitzsimmons** [comment]
@maxxxxxdlp Is this fixed yet?

**Comment by maxxxxxdlp** [comment]
I don't think so, but you could ask @realVinayak who might know more about this (Jason might too)

**Comment by grantfitzsimmons** [comment]
@realVinayak @melton-jason Do we have any plan to fix this? I believe this is not working still.
